src/main/kotlin/app/revanced/patches/youtube/misc/debugging/patch/DebuggingPatch.kt

```diff
@@ -11,16 +11,17 @@ import app.revanced.patches.shared.settings.preference.impl.StringResource
 import app.revanced.patches.shared.settings.preference.impl.SwitchPreference
 import app.revanced.patches.youtube.misc.debugging.annotations.DebuggingCompatibility
 import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
+import app.revanced.patches.all.misc.debugging.patch.SharedDebuggingPatch
 import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
 import org.w3c.dom.Element
 
 @Patch
-@Name("debugging")
-@DependsOn([IntegrationsPatch::class, SettingsPatch::class])
-@Description("Adds debugging options.")
+@Name("debugging-extra")
```

Comment:
Both patch should be called "debugging", or "enable-debugging" if we want to convey what the patch does. The issue would be, that the shared patch would be selected by name as well. Any ideas to that?

Reply:
Maybe we can add an incompatiblePackages annotioation for that [like how the compatiblePackages work](https://github.com/revanced/revanced-patcher/blob/main/src/main/kotlin/app/revanced/patcher/annotation/CompatibilityAnnotation.kt#L11) but i still do not understand what is the problem with renaming the patches

Reply:
I believe naming one patch "enable-android-debugging" and the other "enable-debugging" wouldn't be too off?

Reply:
Thats fine too but i think `debugging` and `debugging-extra` describes its functions better and do not forget that both of them would show up in the manager or when listing patches with revanced-cli

Reply:
It is unclear what "extra" means wheread "enable-android-debugging" explicitly explains, that the debugging capabilities of Android are enabled for the app. For that reason, I'd say it would be better than an arbitrary suffix "extra".